tests/test_supply_curve_wind_dirs.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+"""
+Supply Curve computation integrated tests
+"""
+import os
+import pandas as pd
+from pandas.testing import assert_frame_equal
+import pytest
+
+from reV import TESTDATADIR
+from reV.supply_curve.supply_curve import CompetitiveWindFarms, SupplyCurve
+
+
+TRANS_COSTS = {'line_tie_in_cost': 200, 'line_cost': 1000,
+               'station_tie_in_cost': 50, 'center_tie_in_cost': 10,
+               'sink_tie_in_cost': 100, 'available_capacity': 0.3}
+
+SC_POINTS = os.path.join(TESTDATADIR, 'sc_out/baseline_agg_summary.csv')
+TRANS_TABLE = os.path.join(
+    TESTDATADIR, 'trans_tables/ri_transmission_table.csv')
+MULTIPLIERS = os.path.join(
+    TESTDATADIR, 'trans_tables/transmission_multipliers.csv')
+WIND_DIRS = os.path.join(TESTDATADIR, 'comp_wind_farms/wind_dirs.csv')
+
+
+@pytest.mark.parametrize('downwind', [False, True])
+def test_competitive_wind_dirs(downwind):
+    """Run CompetitiveWindFarms and verify results against baseline file."""
+
+    sc_points = CompetitiveWindFarms.run(WIND_DIRS, SC_POINTS,
+                                         n_dirs=2, sort_on='mean_lcoe',
+                                         downwind=downwind)
+
+    if downwind:
+        baseline = os.path.join(TESTDATADIR, 'comp_wind_farms',
+                                'sc_points_downwind.csv')
+    else:
+        baseline = os.path.join(TESTDATADIR, 'comp_wind_farms',
+                                'sc_points_upwind.csv')
+
+    baseline = pd.read_csv(baseline)
+    assert_frame_equal(sc_points, baseline, check_dtype=False)
+
+
+@pytest.mark.parametrize('downwind', [False, True])
+def test_sc_full_wind_dirs(downwind):
+    """Run the full SC test and verify results against baseline file."""
+
+    sc_out = SupplyCurve.full(SC_POINTS, TRANS_TABLE, fcr=0.1,
+                              sc_features=MULTIPLIERS,
+                              transmission_costs=TRANS_COSTS,
+                              wind_dirs=WIND_DIRS, downwind=downwind)
+
+    if downwind:
+        baseline = os.path.join(TESTDATADIR, 'comp_wind_farms',
+                                'sc_full_downwind.csv')
+    else:
+        baseline = os.path.join(TESTDATADIR, 'comp_wind_farms',
+                                'sc_full_upwind.csv')
+
+    baseline = pd.read_csv(baseline)
+    assert_frame_equal(sc_out, baseline, check_dtype=False)
```

Comment:
Did you verify the results? This gets back to the verification vs. baseline validation discussion. Always a bit nervous to see no code inspection of the results.

Reply:
I don't disagree, but any thoughts on better tests?

Reply:
Could we go through the built SC table and check the upwind/downwind gid's and make sure they are not present in the built SC table? Just confirm that they were indeed competitively eliminated.

Reply:
I like it, will do